pkg/fleet/env/env.go

```diff
@@ -43,6 +44,21 @@ var defaultEnv = Env{
 	},
 }
 
+// ApmLibLanguage is a language defined in DD_APM_INSTRUMENTATION_LIBRARIES en var
+type ApmLibLanguage string
+
+// ApmLibVersion is the version of the library defined in DD_APM_INSTRUMENTATION_LIBRARIES en var
+type ApmLibVersion string
+
+// AsVersionTag returns the version tag associated with the version of the library defined in DD_APM_INSTRUMENTATION_LIBRARIES
+// if the value is empty we return latest
+func (v ApmLibVersion) AsVersionTag() string {
+	if v == "" {
+		return "latest"
+	}
+	return string(v) + "-1"
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏Kinda wonder if we should be smart and not add the -1 at the end if it's already there

Reply:
I'm not sure about that, because we'd need to strip it away in `Env.ToEnv`. I'd rather keep the values in this file as raw as possible

Reply:
Definitely not a blocker and was more to avoid footguns, but if we never display the -1 that works too